tests/__snapshots__/ValidationTest__it_validates_order_with_data_set_dhlforyou_to_France__1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Dit lijkt niet de bedoeling, maar ik heb niet het project binnengehaald nog, is het logisch dat deze foutmelding komt uit deze test?